src/pages/iou/steps/MoneyRequestAmountForm.js

```diff
@@ -0,0 +1,266 @@
+import React, {useEffect, useState, useCallback, useRef} from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import lodashGet from 'lodash/get';
+import _ from 'underscore';
+import styles from '../../../styles/styles';
+import BigNumberPad from '../../../components/BigNumberPad';
+import * as CurrencyUtils from '../../../libs/CurrencyUtils';
+import * as MoneyRequestUtils from '../../../libs/MoneyRequestUtils';
+import Button from '../../../components/Button';
+import * as DeviceCapabilities from '../../../libs/DeviceCapabilities';
+import TextInputWithCurrencySymbol from '../../../components/TextInputWithCurrencySymbol';
+import useLocalize from '../../../hooks/useLocalize';
+import CONST from '../../../CONST';
+
+const propTypes = {
+    /** IOU amount saved in Onyx */
+    amount: PropTypes.number,
+
+    /** Currency chosen by user or saved in Onyx */
+    currency: PropTypes.string,
+
+    /** Whether the amount is being edited or not */
+    isEditing: PropTypes.bool,
+
+    /** Refs forwarded to the TextInputWithCurrencySymbol */
+    forwardedRef: PropTypes.oneOfType([PropTypes.func, PropTypes.object]),
```

Comment:
```suggestion
    forwardedRef: PropTypes.oneOfType([PropTypes.func, PropTypes.shape({current: PropTypes.instanceOf(React.Component)})]),
```

Do we still need to use object here? The prop type does not look correct, if there is still an error please write the ref as a function

Reply:
done